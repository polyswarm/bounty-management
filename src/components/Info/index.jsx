import React, { Component } from 'react';

class Info extends Component {

  render() {
    return (
      <div className='Info'>
      </div>
    );
  }

}

export default Info;
