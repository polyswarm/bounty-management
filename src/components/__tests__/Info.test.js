import React from 'react';
import {render} from 'enzyme';
import {renderToJson} from 'enzyme-to-json';
import Info from '../Info';

it('renders without crashing', () => {
  const wrapper = render(<Info />);
  expect(renderToJson(wrapper)).toMatchSnapshot();
});
