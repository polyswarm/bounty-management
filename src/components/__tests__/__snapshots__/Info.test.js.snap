// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  class="Bounty-Info"
>
  <div
    class="File-List"
  >
    <header
      class="File-List-Header"
    >
      <p
        class="File-List-Title"
      >
        0 Files
      </p>
      <button
        class="Button"
        disabled=""
      >
        - Clear
      </button>
    </header>
    <ul
      class="List"
    >
      
    </ul>
  </div>
</div>
`;
